.codestats.yml

```diff
@@ -0,0 +1,17 @@
+config:
+  ci: 'BITRISE'
+  url: http://codestats.wolox.com.ar/
+  token: <%= ENV['CODE_STATS_TOKEN'] %>
+metrics:
+  slather:
+    enabled: true
+    minimum: 90
```

Comment:
Maybe use a more reachable minimum right now, and then go lifting it accordingly.